rust/src/logger.rs

```diff
@@ -1,3 +1,5 @@
+use std::sync::Once;
+
```

Comment:
🙃 Unwanted newline I suppose.

Reply:
true. after tournament I might start a discussion that it would be actually nice to separate `std` imports from external crates, but it's not a good time to do now :-)